src/main/java/by/onliner/driver/DriverFactory.java

```diff
@@ -0,0 +1,86 @@
+package by.onliner.driver;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import io.github.bonigarcia.wdm.config.DriverManagerType;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.openqa.selenium.edge.EdgeOptions;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.firefox.FirefoxOptions;
+import org.openqa.selenium.opera.OperaDriver;
+import org.openqa.selenium.opera.OperaOptions;
+
+
+public class DriverFactory {
+
+    public enum BrowserType {FIREFOX, CHROME, EDGE, OPERA}
+
+    /***
+     * Chooses a webdriver of a defined type
+     * @param browser
+     */
+    public static WebDriver getDriver(BrowserType browser) {
+            WebDriver driver = null;
```

Comment:
Are you sure initializing with null is a good idea? what will be returned on the 40th line if you pass an unknown browser as an argument here? Would you expect your tests to proceed in such a situation?

Reply:
I need to throw Exeption if my driver couldn't be finded. I will fix that thing